go/host/client_api_eth.go

```diff
@@ -72,3 +73,18 @@ func (api *EthereumAPI) GetTransactionReceipt(_ context.Context, encryptedParams
 	}
 	return gethcommon.Bytes2Hex(encryptedResponse), nil
 }
+
+// GetTransactionCount returns the nonce of the wallet with the given address.
+func (api *EthereumAPI) GetTransactionCount(_ context.Context, encryptedParams common.EncryptedParamsGetTxCount) (string, error) {
+	encryptedResponse, err := api.host.EnclaveClient.GetTransactionCount(encryptedParams)
+	if err != nil {
+		return "", err
+	}
+	// todo: get rid of this hack when we stop supporting unencrypted params for testing purposes
+	// 		(the unencrypted hex number doesn't like to be bytes to hex encoded)
+	if strings.HasPrefix(string(encryptedResponse), "0x") {
```

Comment:
This seemed to me the least painful way to keep transaction injector (unencrypted nonce requests) and wallet_extension_tests (testing with viewing key enc) happy. Would gladly take pointers on a cleaner solution though :)